nemo/collections/asr/parts/submodules/rnnt_decoding.py

```diff
@@ -1006,6 +1045,60 @@ def _get_word_offsets_subwords_sentencepiece(
 
         return word_offsets
 
+    @staticmethod
+    def _get_segment_offsets(
```

Comment:
can you add the same for `ctc_decoding.py`

Reply:
done